src/Field.tsx

```diff
@@ -7,7 +7,7 @@ import {
   FieldValidator,
 } from './types';
 import { useFormikContext } from './FormikContext';
-import { isFunction, isEmptyChildren, isObject } from './utils';
+import { isFunction, isEmptyChildren } from './utils';
 import invariant from 'tiny-warning';
 
 export interface FieldProps<V = any> {
```

Comment:
This isn't your change, but here `FieldProps<V>` appears to be used as both `typeof FieldInputProps<Value>['value']` and `typeof FormikProps<Values>['values']`.